### PR TITLE
Publish Noa, Andzrej and Andres

### PR DESCRIPTION
**Pull request by icougil**

### CODE REVIEW

File: 2022/_data/talks.json [hunk 1]
```diff
@@ -652,21 +652,39 @@
...
+        "17f50b47a57b0354999b00d372ac197a9452a7c1"
+      ]
+    },
+    {
+      "id": "62505f60aa23f0085462dda5",
+      "title": "Modern Java microservices in the cloud",
+      "languages": [
+        "eng"
+      ],
+      "abstract": "It's 2022, long gone are the days of ceremony and Java development should be productivity, simplicity.\n\nLet's look at how we make it happen, in context of startup going FAST.\nI will describe the architectural approach we use stack we use, and what, sometimes unorthodox, technology and design choices we have made, and their consequences. Expect: code samples, why does our stack look the way it looks AND… what have we learnt in the process.\n\nIt's all going to be Revolut-based!",
```
Comment (abelsromero) [inline 2022/_data/talks.json:677]: El abstract no está formateado, ojo q se haya generado en html 
image

**Review by abelsromero — CHANGES_REQUESTED** [review]

**Review by anyulled — APPROVED** [review]
I understand that the code deletion is regarding some duplicated code, right?

**Comment by icougil** [comment]
> I understand that the code deletion is regarding some duplicated code, right?

yep, you can compare it side by side to double check it, but yes it is basically because of the things that git does (don't know why the content it is moved)

File: 2022/_data/talks.json (hunk 1 above)
Comment (icougil) [reply 2022/_data/talks.json:677]: macho... esto ya está en main...

Comment (icougil) [reply 2022/_data/talks.json]: hecho

**Comment by abelsromero** [comment]
> yep, you can compare it side by side to double check it, but yes it is basically because of the things that git does (don't know why the content it is moved)

My theory is at some point some manual edit was done or some rebase from gh-pages.

**Comment by icougil** [comment]
@abelsromero do the changes look good now?

**Pull request merged**